Bibliotheque nationale de France.js

```diff
@@ -1,7 +1,7 @@
 {
 	"translatorID": "47533cd7-ccaa-47a7-81bb-71c45e68a74d",
 	"label": "Bibliothèque nationale de France",
-	"creator": "Florian Ziche",
+	"creator": "Florian Ziche, Sylvain Machefert",
 	"target": "^https?://[^/]*catalogue\\.bnf\\.fr",
```

Comment:
would be good idea to take the opportunity to adjust the regexp. `"^https?://([^/]+\\.)?catalogue\\.bnf\\.fr/"`